raiden-dapp/src/services/raiden-service.ts

```diff
@@ -12,14 +12,18 @@ import {
 import { BigNumber } from 'ethers/utils';
 import { Zero } from 'ethers/constants';
 import { filter, first } from 'rxjs/internal/operators';
+import { ConfigProvider } from './config-provider';
 
 export default class RaidenService {
   private _raiden?: Raiden;
   private store: Store<RootState>;
 
-  private static async createRaiden(provider: any): Promise<Raiden> {
+  private static async createRaiden(
+    provider: any,
+    account?: string
+  ): Promise<Raiden> {
     try {
-      return await Raiden.create(provider, 0, window.localStorage);
+      return await Raiden.create(provider, account || 0, window.localStorage);
```

Comment:
I think this would be more descriptive if you had `account: string | number = 0` on the method parameter, and then just pass `account` to `create`, so it will default to `0` if omitted but still suport either `string | number`